modules/rostests/apitests/shell32/ShellExecuteEx.cpp

```diff
@@ -14,6 +14,7 @@
 #include <stdio.h>
 #include <strsafe.h>
 #include <versionhelpers.h>
```

Comment:
Is this include still needed?

Reply:
versionhelpers? Probably not but it does not hurt anyone?

Reply:
If removing unnecessary header would make the final executable smaller, it would make @JoachimHenze happy.

Reply:
Yes, it is needed for ```IsWindowsVistaOrGreater()```.

Reply:
1) All the functions in this header are inline, should add no code if they are not used.

2) The header is used by one of the other files in the shell32 test so any theoretical global/static data would still be there.

I'll remove the header from this file anyway before I merge.

Reply:
> Yes, it is needed for `IsWindowsVistaOrGreater()`.

Indeed! Somehow, I missed that other occurrence. Thanks.